docs/customization/customizing-your-storefront.mdx

```diff
@@ -0,0 +1,170 @@
+---
+sidebar_position: 1
+---
+
+# Customizing your storefront
+
+In this guide, we're going to show you an example of how to customize your storefront. In
+particular, we're going to add a hero image to the hero section of the home page:
+
+![Hero image placeholder](/img/customizing-your-storefront/hero-image-placeholder.png)
+
+## Installing the frontend
+
+With Solidus 3.2, installing Solidus would automatically install[SolidusStarterFrontend][], which is
+the new frontend solution for Solidus. As a Rails application template, it copies all the Solidus
```

Comment:
I'd move this reference to 3.2 in an info box somewhere in the page. Something like "Before 3.2, we used to do XYZ, here's the link to the legacy solution, etc etc". For the majority of the new users, this information will just be confusing and they should see a clear main way of doing things without distractions. WDYT?

Reply:
@kennyadsl I'll try this out:

* Merge Intro and "Installing the frontend" section.
* Move Solidus installation background to info box.

TODO